cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Replica.java

```diff
@@ -130,6 +130,14 @@ AggregatedMetricValues makeFollower() {
     // Remove leadership from the replica.
     setLeadership(false);
 
+    return leaderLoadDelta(true);
+  }
+
+  public AggregatedMetricValues leaderLoadDelta() {
+    return leaderLoadDelta(false);
+  }
+
+  private AggregatedMetricValues leaderLoadDelta(boolean updateLoad) {
```

Comment:
In a comment or JavaDoc, can we clarify the role of this function -- i.e. it looks like a `getter`, but it is potentially clearing the load.